### PR TITLE
Implement "Remember my Organization" in multiple-ldap

### PR DESCRIPTION
**Pull request by aguvillalba**
This new feature allows the users to have a checkbox in the login form
which works in the same way than "Remember my username" but for the
organization selected in the dropdown
Implements feature requested in issue: Allow "Remember organization" for multiple-ldap authsource #703

### CODE REVIEW

File: config-templates/authsources.php [hunk 1]
```diff
@@ -354,6 +354,11 @@
         //'remember.username.enabled' => FALSE,
         //'remember.username.checked' => FALSE,
 
+        // Give the user an option to save their organization choice for future login
+		// attempts. And when enabled, what should the default be, checked or not.
+		//'remember.organization.enabled' => false,
+        //'remember.organization.checked' => false,
```
Comment (tvdijen) [inline config-templates/authsources.php]: Your white-spaces are off here

File: modules/core/lib/Auth/UserPassOrgBase.php [hunk 2]
```diff
@@ -137,6 +162,21 @@ public function getRememberUsernameChecked() {
 		return $this->rememberUsernameChecked;
 	}
 
+    /**
+     * Getter for the authsource config option remember.organization.enabled
+     * @return bool
+     */
+	public function getRememberOrganizationEnabled() {
```
Comment (tvdijen) [inline modules/core/lib/Auth/UserPassOrgBase.php]: Following PSR-2 code guidelines, the `{` should go on the next line

File: modules/core/lib/Auth/UserPassOrgBase.php [hunk 3]
```diff
@@ -137,6 +162,21 @@ public function getRememberUsernameChecked() {
...
+		return $this->rememberOrganizationEnabled;
+	}
+
+	/**
+	 * Getter for the authsource config option remember.organization.checked
+	 * @return bool
+	 */
+	public function getRememberOrganizationChecked() {
```
Comment (tvdijen) [inline modules/core/lib/Auth/UserPassOrgBase.php]: Same here

**Comment by tvdijen** [comment]
@thijskh How about we merge this into the Xnew-ui branch?

**Comment by jaimeperez** [comment]
I agree, I think it makes sense to add this functionality directly to the new user interface.

**Comment by aguvillalba** [comment]
Solved the PSR-2 mentioned issues.
Regarding to migrate to Xnew-ui, ok! I can try it, but I am not sure when it can be finished (busy with other stuff). I will definitely try.

**Comment by jaimeperez** [comment]
¡Gracias Agustín! 😉

**Comment by tvdijen** [comment]
I have put a little effort in the Twig-template and then merged this into the Xnew-ui branch

**Comment by thijskh** [comment]
Afraid not, this will be in 1.17. Updated milestone.

**Comment by tvdijen** [comment]
It only exists in master right now

**Comment by aguvillalba** [comment]
So, what should I do then?

**Comment by thijskh** [comment]
Wait until 1.17 is released?

**Pull request closed without merging**